Compilation error

On CI https://travis-ci.org/github/ndmitchell/hlint/jobs/667865822 I'm getting:

```
Building executable 'refactor' for apply-refact-0.6.0.0..
[1 of 6] Compiling Paths_apply_refact ( dist/build/refactor/autogen/Paths_apply_refact.hs, dist/build/refactor/refactor-tmp/Paths_apply_refact.o )
[2 of 6] Compiling Refact.Utils     ( src/Refact/Utils.hs, dist/build/refactor/refactor-tmp/Refact/Utils.o )
[3 of 6] Compiling Refact.Fixity    ( src/Refact/Fixity.hs, dist/build/refactor/refactor-tmp/Refact/Fixity.o )
src/Refact/Fixity.hs:59:27: error:
    Ambiguous occurrence ‘noExt’
    It could refer to either ‘HsExtension.noExt’,
                             imported from ‘HsExtension’ at src/Refact/Fixity.hs:12:1-18
                          or ‘Language.Haskell.GHC.ExactPrint.Types.noExt’,
                             imported from ‘Language.Haskell.GHC.ExactPrint.Types’ at src/Refact/Fixity.hs:16:1-73
   |
59 |   = return $ L loc (OpApp noExt e1 op2 e2)
   |                           ^^^^^
```

Is the issue a change to ghc-exactprint-0.6.3?

--- a/apply-refact.cabal
+++ b/apply-refact.cabal
@@ -32,7 +32,7 @@ library
   GHC-Options: -Wall
   build-depends: base >=4.8 && < 5
                , refact >= 0.2
-               , ghc-exactprint >= 0.6.2
+               , ghc-exactprint >= 0.6.3
                , ghc >= 8.8.1
                , containers
                , syb
@@ -59,7 +59,7 @@ executable refactor
   ghc-options: -Wall -fno-warn-unused-do-bind
   build-depends: base >= 4.8 && < 5
                , refact >= 0.2
-               , ghc-exactprint >= 0.6.2
+               , ghc-exactprint >= 0.6.3
                , ghc >= 8.8.1
                , containers
                , syb
@@ -92,7 +92,7 @@ Test-Suite test
                      , tasty-expected-failure
                      , base < 5
                , refact >= 0.2
-               , ghc-exactprint >= 0.5.7.0
+               , ghc-exactprint >= 0.6.3
                , ghc >= 8.8.1
                , containers
                , syb

--- a/src/Refact/Apply.hs
+++ b/src/Refact/Apply.hs
@@ -36,7 +36,7 @@ import Data.Generics.Schemes
 
 import HsExpr as GHC hiding (Stmt)
 import HsImpExp
-import HsSyn hiding (Pat, Stmt)
+import HsSyn hiding (Pat, Stmt, noExt)
 import SrcLoc
 import qualified GHC hiding (parseModule)
 import qualified OccName as GHC

--- a/src/Refact/Fixity.hs
+++ b/src/Refact/Fixity.hs
@@ -9,7 +9,7 @@ import Refact.Utils
 import BasicTypes (Fixity(..), defaultFixity, compareFixity, negateFixity, FixityDirection(..), SourceText(..))
 import HsExpr
 import RdrName
-import HsExtension
+import HsExtension hiding (noExt)
 import OccName
 import Data.Generics hiding (Fixity)
 import Data.Maybe
@@ -162,4 +162,3 @@ infix_  = fixity InfixN
 -- Internal: help function for the above definitions.
 fixity :: FixityDirection -> Int -> [String] -> [(String, Fixity)]
 fixity a p = map (,Fixity (SourceText "") p a)
-

